services/user-tenant-service/src/main/java/com/sourcefuse/jarc/services/usertenantservice/repository/GroupRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.sourcefuse.jarc.services.usertenantservice.repository;
+
+import com.sourcefuse.jarc.services.usertenantservice.dto.Group;
+import java.util.UUID;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface GroupRepository extends JpaRepository<Group, UUID> {}
```

Comment:
This needs to be implemented using soft delete repo @harshadk-sourcefuse guide him